HW_1/triangle.py

```diff
@@ -0,0 +1,25 @@
+'''Треугольник существует только тогда, когда сумма любых двух его сторон больше третьей. 
+Дано a, b, c - стороны предполагаемого треугольника. 
+Требуется сравнить длину каждого отрезка-стороны с суммой двух других. 
+Если хотя бы в одном случае отрезок окажется больше суммы двух других, то треугольника с такими сторонами не существует. 
+Отдельно сообщить является ли треугольник разносторонним, равнобедренным или равносторонним.
+'''
+
+# Ввод сторон треугольника
+a = float(input("Введите длину стороны a: "))
+b = float(input("Введите длину стороны b: "))
+c = float(input("Введите длину стороны c: "))
+
+# Проверка условий существования треугольника
+if a + b > c and a + c > b and b + c > a:
+    print("Треугольник существует")
+
+    # Проверка типа треугольника
+    if a == b and b == c:
+        print("Треугольник равносторонний")
+    elif a == b or b == c or a == c:
+        print("Треугольник равнобедренный")
+    else:
+        print("Треугольник разносторонний")
```

Comment:
Можно немного сократить условие
```suggestion
    elif a != b != c:
        print("Треугольник разносторонний")
    else:
        print("Треугольник равнобедренный")
```